inginious/frontend/plugins/adaptative/README.rst

```diff
@@ -0,0 +1,3 @@
+
+[![Rdoc](http://www.rdocumentation.org/badges/version/catR)](http://www.rdocumentation.org/packages/catR)
```

Comment:
We will not add new plugins in the main INGInious repository without a very good reason.